euth/comments/apps.py

```diff
@@ -1,5 +0,0 @@
-from django.apps import AppConfig
-
-
-class CommentsApiConfig(AppConfig):
-    name = 'comments'
```

Comment:
From Django's [Applications Documentation](https://docs.djangoproject.com/en/1.8/ref/applications/#configuring-applications):

> New applications should avoid default_app_config. Instead they should require the dotted path to the appropriate AppConfig subclass to be configured explicitly in INSTALLED_APPS.

This means:

1. An app should have an `apps.py` module
2. `INSTALLED_APPS` should use the dotted path to an `AppConfig` class in this module
3. An app should not use `default_app_config`, it only exists for backwards compatibility